doc/lua_api.txt

```diff
@@ -1692,8 +1679,8 @@ Metadata
 Node Metadata
 -------------
 The instance of a node in the world normally only contains the three values
-mentioned in "Nodes". However, it is possible to insert extra data into a
-node. It is called "node metadata"; See `NodeMetaRef`.
+mentioned in "Node paramtypes". However, it is possible to insert extra data
```

Comment:
"Nodes" is correct, this is referring to
`{name="name", param1=num, param2=num}`

Reply:
Now fixed.